src/Microsoft.OData.Edm/Csdl/CsdlReader.Json.cs

```diff
@@ -0,0 +1,201 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CsdlReader.Json.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.OData.Edm.Csdl.CsdlSemantics;
+using Microsoft.OData.Edm.Csdl.Parsing;
+using Microsoft.OData.Edm.Csdl.Parsing.Ast;
+using Microsoft.OData.Edm.Validation;
+
+namespace Microsoft.OData.Edm.Csdl
+{
+    /// <summary>
+    /// A delegate used to set the referenced CSDL-JSON reader.
+    /// </summary>
+    /// <param name="uri">The reference Uri.</param>
+    /// <param name="skip">out a boolean value that indicates whether to skip parsing the input uri.</param>
+    /// <returns>The corresponding <see cref="Utf8JsonReader"/>.</returns>
+    /// <remarks>
+    /// Utf8JsonReader is a ref struct, so we can't distiguish which Uri you want to parse, which Uri want to skip.
+    /// So, use an out boolean value to indicate whether to skip parsing the input reference Uri.
+    /// </remarks>

```

Comment:
probably better as a comment than a doc remark (which appears in generated documentation)

Reply:
Moved to <summary></summary>

---
In reply to: [429388470](https://github.com/OData/odata.net/pull/1775#discussion_r429388470) [](ancestors = 429388470)